packages/react-app/src/components/Header.tsx

```diff
@@ -5,12 +5,16 @@ interface Props {
   userNft: number;
   isParcelCountRead: boolean;
   userAddress?: string;
+  setShowingOwnedPlot: any;
 }
-export default function Header({ connectWallet, userNft, isParcelCountRead, userAddress }: Props) {
+export default function Header({ connectWallet, userNft, isParcelCountRead, userAddress, setShowingOwnedPlot }: Props) {
+  console.log(isParcelCountRead)
+  console.log(userAddress)
+  console.log(userNft)
   return (
     <div className="connect-wallet-section header border-b primary-font text-xl flex h-16 lg:px-4 items-center">
       {isParcelCountRead && userAddress && (
-        <span className="bg-gray-1 opacity-75 text-lg secondary-font mr-5">
+        <span className="bg-gray-1 opacity-75 text-lg secondary-font mr-5" onClick={() => setShowingOwnedPlot(true)}>
           {userNft > 0 ? `YOUR PLOTS (${userNft})` : `No plots`}
```

Comment:
What will happen if user clicks "No plots" this link supposed to be added only to  Your Plots title, not No plots.